app/views/providers/limitations/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= page_template(
+      page_title: t('.h1-heading')
+    ) %>
+
+<% if @legal_aid_application.proceeding_types.present? %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m">
+        <%= t('.h2-heading', count: pluralize(@legal_aid_application.proceeding_types.size, t('.h2-heading'))) %>
+      </h2>
+      <div class="selected-proceeding-types">
+        <%= render partial: 'shared/forms/proceedings_types/proceeding_type', collection: @legal_aid_application.proceeding_types, locals: { translation_path: 'providers.limitations.show' } %>
+      </div>
+
+      <div class="govuk-!-padding-bottom-6"></div>
+
+      <p class="govuk-body">
+        <%= t('.form_of_service.html') %> <%= t('.form_of_service_type') %> <%= content_tag(:br) %>
```

Comment:
```<%= content_tag(:br) %>``` :scream: 

Just `<br>`

Having said that, there is almost always a better way of separating markup into separate lines. This looks like a [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) to me. So:
```
<dl>
  <dt><%= t('.form_of_service.html') %></dt>
  <dd><%= t('.form_of_service_type') %></dd>

  <dt><%= t('.incurrable_costs.html') %></dt>
  <dd><%= t('.incurrable_costs_value') %></dd>
  ....
```
Then use CSS (and perhaps some gov-uk classes) to get the formatting right

Reply:
Julien's paragraphs works too if you prefer.